lib/utils/string-utils.js

```diff
@@ -12,6 +13,15 @@ module.exports = {
     validateSyntax
 };
 
+function getParamNames(func) {
+    const STRIP_COMMENTS = /((\/\/.*$)|(\/\*[\s\S]*?\*\/))/mg;
+    const ARGUMENT_NAMES = /([^\s,]+)/g;
```

Comment:
We have a REGEX_VALIDATIONS in constants.js, should we try to put regex in one place?